com.unity.netcode.gameobjects/Tests/Runtime/NetworkShowHideTests.cs

```diff
@@ -134,7 +134,9 @@ public IEnumerator NetworkShowHideTest()
             m_NetSpawnedObject2.Spawn();
             m_NetSpawnedObject3.Spawn();
 
-            yield return new WaitForSeconds(0.1f);
+            // Using frame count as opposed to time to avoid random failures
+            var waitUntilFrameNumber = Time.frameCount + 10;
+            yield return new WaitUntil(() => Time.frameCount >= waitUntilFrameNumber);
```

Comment:
I put a different fix for this in https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1062 . Even though your change improves things, I came to the conclusion that anything time-based is brittle. The other change times out after a while, but continues early when the show/hide happens quickly. I'd suggest leaving out those changes (line 137-139, 149-150, and 157-159). If not, no big deal, but the merge with the other will have them removed anyway :-)

Reply:
Yeah, I figured when you merged your changes they would go away or I would remove them and accept your changes.
Without those changes my builds will randomly fail in Yamato...so...how about we wait until your PR is merged?
I did a quick review of your PR, looked relatively clean so I approved it if you want to merge that before I merge this...then I would be happy to merge your changes into this PR.

Reply:
Merged!